src/ButtonPanel.ts

```diff
@@ -44,7 +51,7 @@ export default class ButtonPanel {
     this._isHidden = false
   }
 
-  mount(element) {
+  mount(element: Element): void {
```

Comment:
The `void` is already defined in the `ButtonPanelMethods` :wink:

Reply:
You're right! I'm on it!